### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by usamahjundia**
Added the description for the feature in issue #1 . Is there any significance of reporting the last time a path was removed from history that i can add?

### CODE REVIEW

**Comment by imambungo** [comment]
Yeah, you can add something like "This may help you decide the history size that suits you"
with "history size" is a link to [#um-only-10](https://github.com/imambungo/dirjump#um-only-10).

**Comment by usamahjundia** [comment]
I have updated the description to also hint the user that it may help adjusting the history size, and refer to the link mentioned above.

File: README.md [hunk 1]
```diff
@@ -18,6 +18,7 @@ that still saved after exiting the terminal. A directory is considered to be use
 
 - Automatically list all files in the directory you jumped to if it contains 30 or fewer files. You can [disable this feature](#dont-list-files-after-jump) if you like.
 - Home or `~/` is never added to the history (we can just type `cd`).
+- Shows the last time a path was removed from the history. This information may help you adjust the history size to better suit your need. To change the history size, refer to [this section](https://github.com/imambungo/dirjump/blob/master/dirjump#um-only-10)
```
Comment (imambungo) [inline README.md]:
```suggestion
- Shows the last time a path was removed from the history. This information may help you adjust the history size to better suit your need. To change the history size, refer to [this section](#um-only-10)
```

**Review by imambungo — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (imambungo) [reply README.md]: We don't need to specify the URL or path if we refer to the file itself.

**Comment by usamahjundia** [comment]
I have updated the readme file.

**Review by imambungo — APPROVED** [review]
LGTM, thanks for the PR :+1:

**Pull request merged**